tests/Extensions/Hosting.Services.UnitTests/OmexHostedServiceTests.cs

```diff
@@ -44,24 +40,22 @@ public async Task StartAsync_ProperlyCanceled()
 		[TestMethod]
 		public async Task StartAsync_HandlesExceptions()
 		{
-			MockRunner runnerMock = new MockRunner(t => Task.Run(() => throw new Exception("Totaly valid exeption")));
-			MockLifetime lifetimeMock = new MockLifetime();
+			MockRunner runnerMock = new MockRunner(t => Task.Run(async () =>
+				{
+					await Task.Delay(5);
```

Comment:
>Delay [](start = 16, length = 5)

I'd suggest adding `.ConfigureAwait(false)` to here and the outer loop for better correspondence between this and the product code.

Same for another few `await`s below

Reply:
Makes sense, added for all awaits in this file, thank you